tests/publisher/tests_reserved_name_dispute.py

```diff
@@ -0,0 +1,136 @@
+import responses
```

Comment:
I think some of this tests already existed in `tests/publisher/test_register_name_dispute.py`?